### PR TITLE
resource_rule_policy

### PR DESCRIPTION
**Pull request by volosatik**
closes CMPT-33810
closes CMPT-33809

### CODE REVIEW

File: examples/lb/l7policy/base-pool.tf [hunk 1]
```diff
@@ -0,0 +1 @@
+../pool/main.tf
```
Comment (ftersin) [inline examples/lb/l7policy/base-pool.tf:1]: is it required for policy?

File: examples/lb/l7policy/main.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+resource "vkcs_lb_l7policy" "app_http_redirect" {
+	name         = "http-tf-example"
+	description  = "Policy for http member/members testing"
+	action       = "REDIRECT_TO_URL"
+	position     = 1
+	listener_id  = vkcs_lb_listener.app_http.id
+	redirect_url = "http://www.example.com"
```
Comment (ftersin) [inline examples/lb/l7policy/main.tf:7]: weird

File: examples/lb/l7policy/main.tf [hunk 3]
```diff
@@ -0,0 +1,8 @@
+resource "vkcs_lb_l7policy" "app_http_redirect" {
```
Comment (ftersin) [inline examples/lb/l7policy/main.tf:1]: l7 is http

File: examples/lb/l7policy/main.tf [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+	name         = "http-tf-example"
+	description  = "Policy for http member/members testing"
```
Comment (ftersin) [inline examples/lb/l7policy/main.tf:3]: really?

File: examples/lb/l7policy/base-pool.tf (hunk 1 above)
Comment (volosatik) [reply examples/lb/l7policy/base-pool.tf:1]: now it is

File: examples/lb/l7policy/main.tf (hunk 2 above)
Comment (volosatik) [reply examples/lb/l7policy/main.tf]: no longer redirect to url

File: examples/lb/l7policy/main.tf (hunk 3 above)
Comment (volosatik) [reply examples/lb/l7policy/main.tf]: fixed

File: examples/lb/l7policy/main.tf (hunk 4 above)
Comment (volosatik) [reply examples/lb/l7policy/main.tf]: fixed

File: examples/lb/l7policy/main.tf [hunk 5]
```diff
@@ -0,0 +1,8 @@
+resource "vkcs_lb_l7policy" "app_redirect" {
+	name         = "http-tf-example"
+	description  = "Policy for tf lb testing"
+	action       = "REDIRECT_TO_POOL"
+	position     = 1
+	listener_id  = vkcs_lb_listener.app_http.id
+	redirect_pool_id = vkcs_lb_pool.http.id
```
Comment (ftersin) [inline examples/lb/l7policy/main.tf:7]: format?

**Review by ftersin — APPROVED** [review]

**Pull request merged**